src/python/qequlacs/simulator.py

```diff
@@ -43,6 +49,9 @@ def run_circuit_and_measure(self, circuit, **kwargs):
         return Measurements(bitstrings)
 
     def get_expectation_values(self, circuit, qubit_operator, **kwargs):
+        """TODO: Needs a good explanation what is  it and how
```

Comment:
@mstechly here I really need to get better understanding

Reply:
Looking for some good educational materials on expectation values.
Also, the difference between `get_expectation_values` and `get_exact_expectation_values` is that the latter one uses wavefunction, so an exact representation of the quantum state that's impossible to do on the real device, but saves you a lot of time if you use it with a simulator.